scripts/pagerduty_notify.rb

```diff
@@ -82,6 +86,16 @@ def check_arguments(arglist)
 ap output_data if(ENV['debug'])
 #JSON.dump(output_data)
 
+disable_flag=false
+
+if ENV['disable_communications']
+  disable_flag=$store.substitute_string(ENV['disable_communications'])
+  if disable_flag.downcase=='yes' or disable_flag.downcase=='true' or disable_flag.downcase=='disable'
```

Comment:
well, yeah I guess true/false should be enough given that we'd have to translate them anyway.  I just have a habit of forgetting them, so put in a couple of permuations.  'TRUE' is the value that is set if <disable_communications/> were to be set in the route config, and 'yes' is the value that I would naturally put in the route.... waffling now....